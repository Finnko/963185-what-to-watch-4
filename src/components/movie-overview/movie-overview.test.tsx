import * as React from 'react';
import * as renderer from 'react-test-renderer';
import MovieOverview from './movie-overview';

const props = {
  genre: `drama`,
  director: `Wes Bos`,
  rating: 5.5,
  ratingCount: 199,
  actors: [`Ozzy Osbourne`, `Emilia Clarke`],
  description: `Something, something Else`,
};

describe(`MovieOverview component render correctly`, () => {
  it(`Should MovieOverview component render correctly`, () => {
    const tree = renderer
      .create(
          <MovieOverview
            {...props}
          />
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
