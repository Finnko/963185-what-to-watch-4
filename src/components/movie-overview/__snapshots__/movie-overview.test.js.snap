// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieOverview component render correctly Should MovieOverview component render correctly 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      5.5
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Good
      </span>
      <span
        className="movie-rating__count"
      >
        199
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      Something, something Else
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        Wes Bos
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring:
         Ozzy Osbourne, Emilia Clarke
      </strong>
    </p>
  </div>,
]
`;
